src/regdesk/communication_manager.js

```diff
@@ -58,6 +73,14 @@ export class CommunicationManager {
     return results;
   }
 
+  /**
+   * Toggle the state of accepting payments.
+   * @param {CustomEvent} e - The event object
+   */
+  #togglePayments(e) {
```

Comment:
Not sure if this is triggering a re-draw or not. If someone is on the screen and toggles the button, they may need to "refresh" the page to see the updates.

Reply:
It's intentionally not, as the only place we check the payment status is on state entry. The expectation is nobody should be flipping this setting on the fly, just once per day on setup.